mapper/thin_edge_json/src/json.rs

```diff
@@ -141,20 +141,14 @@ pub fn parse_str<T: GroupedMeasurementVisitor>(
     match &thin_edge_obj {
         JsonValue::Object(thin_edge_obj) => {
             for (key, value) in thin_edge_obj.iter() {
-                if key.contains('\\') {
-                    return Err(ThinEdgeJsonError::InvalidThinEdgeJsonKey {
-                        key: String::from(key),
-                    }
-                    .into());
-                }
```

Comment:
Why this has been removed?

To be honest, I have no clue on the purpose of this very specific check, but removing it seems unrelated to that PR.